### ISSUE TITLE
Feature GB Cost Calculator

### ISSUE THREAD

**Issue by vqt123** [dsiekiera/foe-helfer-extension#1162]
Currently the GB cost calculator tells you in BLUE how much you gain to profit, and this is the number I use to know how much more I need to contribute to get to a 1.9 contribution. HOWEVER

since I have a 90 arc.. I need to click between my Arc % and the 90% and do mental math and subtract the difference from the total profit to know exactly how much I need left to contribute. 

I'm not exactly sure how to best solve this problem, just wanted to surface an area of possible improvement. Total profit, although nice, I find not quite as useful as a "how much left do i need to contribute to reach 1.9"

**Comment by scibuff** [comment]
I'm not sure I follow. On the right you have 90% Arc bonus and on the left you can set the % to any number you like, the provided buttons have 85%, 90% and your current arc level (in my example 92%).

So, if I wish to contribute to a guildie at 1.92 I set the "friendly invest" to 92% and I just the cost column from the left side.

![image](https://user-images.githubusercontent.com/722049/83914575-c1a23e80-a771-11ea-9395-5aa2275e86de.png)

Sorry if I misunderstood, please let me know if you are after something different

**Comment by vqt123** [comment]
So the problem is I want to contribute 1.90 and I would like to not have to manually do the subtraction to know exactly how much more to add . This is especially true when filling spots for cycling quests. 

Example someone asks for a 1.9 lock

Total is 1037

I cycle 79 FP 10 times for meaning i have 790 invested so far

Exactly how many more FP do i need to add to get to 1037? 

Would be nice to not have to mentally do 1037 - 790

Right now What i do is look at the Profit, click back and forth between 91% and 90%.. and subtract that difference from the profit to know how much is remaining

**Comment by vqt123** [comment]
![1](https://user-images.githubusercontent.com/3741649/83931315-01fdbe80-a76a-11ea-9ca3-ca5910371622.PNG)
![2](https://user-images.githubusercontent.com/3741649/83931317-04f8af00-a76a-11ea-99fe-deb7e861ab85.PNG)

**Comment by vqt123** [comment]
Example:

I'm on my last RQ and want to simply finish locking my guild mate for 1.9

Exactly how many FP do i put on to complete the 1.9 lock?

Couple ways
Method 1 - Sorta sucky
Do the subtraction of 1834 - 1738 = 96 (I HOPE?!?!) in your head

Method 2 - Not quite as sucky
- Click back forth 91 and 90 % noting the difference (much easier math) 1843 - 1834 = 9
- Subtract 9 from profit  105 - 9 = 96
- Donate 96

Does that make sense?

TLDR: Would be great to simply have the Delta between Total and current so the player doesn't have to do the subtraction in their head to complete the 1.9 donation.

**Comment by scibuff** [comment]
There is a tooltip which appears when you hold the mouse over the cost column. The tooltip shows the difference between what you've paid and what needs to be paid based on the friendly invest value:

![image](https://user-images.githubusercontent.com/722049/83934488-bd961100-a7b1-11ea-830d-ba6f47b258e1.png)

p.s. you really cannot do a simple subtraction in your head?

**Comment by vqt123** [comment]
@scibuff i never realized there was a tooltip!! and Scibuff i know it's laughable, doing subtraction in my head haha. I'm also a software engineer working in the games industry! But please be sympathetic when I'm filling 100+ 1.9s a day and want to try to avoid mistakes and all that mental energy!

**Comment by scibuff** [comment]
@vqt123 Yeah I know :) . To be honest, doing the recurring quests through BE threads is not the best idea because of sniping (ever since you do;nt have to add FPs by 10s). I find that adding all FPs in one go motivates me to take a lot more slots than if I do recurring quests on 2-3 GBs a day. Taking 1st and 2nd BE (90%) slots with 91-92% Arc makes a good earning as well...

**Comment by vqt123** [comment]
@scibuff  well it's actually much safer thanks to your tool. What I do is call the spot, start cycling until i see FP left amount is at a point that is snipable. At which point i quickly add the amount i need to make sure it's not snipable. And continue cycling until the very last one, in which i will now use the tooltip to add the remaining.
